packages/vm/src/index.ts

```diff
@@ -126,6 +128,29 @@ export interface VMOpts {
    * pointing to a Shanghai block: this will lead to set the HF as Shanghai and not the Merge).
    */
   hardforkByTD?: BNLike
+
+  /**
+   * Override or add custom opcodes to the VM instruction set
+   * These custom opcodes are EIP-agnostic and are always statically added
+   * To delete an opcode, add an entry of format `{opcode: number}`. This will delete that opcode from the VM.
+   * If this opcode is then used in the VM, the `INVALID` opcode would instead be used.
+   * To add an opcode, add an entry of the following format:
+   * {
+   *    // The opcode number which will invoke the custom opcode logic
+   *    opcode: number
+   *    // The name of the opcode (as seen in the `step` event)
+   *    opcodeName: string
+   *    // The base fee of the opcode
+   *    baseFee: number
+   *    // If the opcode charges dynamic gas, add this here. To charge the gas, use the `i` methods of the BN, to update the charged gas
```

Comment:
Just a note but this will need to be updated when we rebase this change on develop to reference BigInts and not BN since the concept of `i` methods doesn't exist and can easily trip people up since bigints aren't objects that can be modified in place.

Reply:
Good point we have to keep this in mind when we rebase develop! :)